### PR TITLE
Gui 20 create compose file

### PR DESCRIPTION
**Pull request by AndreyKoltsov1997**
Create docker-compose.ymlwhich pulls mangoes image from docker hub.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: '3'
+
+services:
+  mangoose-console:
+     image: emcmongoose/mongoose-console:0.1.0
+  grafana:
+     image: grafana/grafana
+  prometheus:
+    image: prom/prometheus
```
Comment (akurilov) [inline docker-compose.yml]: please add a newline at the end of the file

**Comment by akurilov** [comment]
The services are unaccessible outside the containers via the default ports. Need to expose the console/prometheus/grafana ports

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+services:
+  mangoose-console:
+     image: emcmongoose/mongoose-console:0.1.0
```
[the remark this answers is not in the document]
Comment (AndreyKoltsov1997) [reply docker-compose.yml]: In order to implement the requirement, I've added "latest" tag to the latest image builded with grade via 'buildImage' command (commit: https://github.com/emc-mongoose/console/pull/21/commits/fe1e81dae23b6ba77ac5e283bb379d084649f808). 
Yet, I don't think it's a good idea to place "latest" on every builded image. Consider create issue to implement something like "buildImageAsLatest' command at build.gradle in the future, may be?

**Comment by AndreyKoltsov1997** [comment]
> The services are unaccessible outside the containers via the default ports. Need to expose the console/prometheus/grafana ports

Default ports has been added.

File: docker-compose.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (akurilov) [reply docker-compose.yml]: Please use the env var for the image tag. It is latest if not specified explicitly and makes possible to run the console image with a custom image tag.

Also, please fix the grafana and prometheus versions using the latest numeric ones. We should use the same (fixed) versions of our 3-rd party components.

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  mangoose-console:
+     image: emcmongoose/mongoose-console
+     ports:
+     - "${MONGOOSE_PORT}"
+  grafana:
+     image: grafana/grafana
```
Comment (akurilov) [inline docker-compose.yml]: please use the tag "5.3.1" for this

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+     ports:
+     - "${GRAFANA_PORT}"
+  prometheus:
+    image: prom/prometheus
```
Comment (akurilov) [inline docker-compose.yml]: please use the tag "v2.4.3" for this

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  mangoose-console:
+     image: emcmongoose/mongoose-console
```
Comment (akurilov) [inline docker-compose.yml]: please use an env var for the image tag

File: .env [hunk 6]
```diff
@@ -0,0 +1,3 @@
+MONGOOSE_PORT=80
```
Comment (akurilov) [inline .env]: AFAIK we decided to use another port for this outside of the container

Comment (AndreyKoltsov1997) [reply .env]: it's a port **inside** the container and it binds to a host port (38888 as an example in the picture below) by docker itself. 
Inside the container, NGINX is using this port (80). 
image

Comment (akurilov) [reply .env]: Ok, but it should be "NGINX_PORT" and not "MONGOOSE_PORT"

**Pull request merged**